packages/scikit-image/patches/no-openmp.patch

```diff
@@ -0,0 +1,10 @@
+--- a/setup.py	2019-04-01 22:30:11.000000000 +0000
++++ b/setup.py	2020-12-16 06:21:52.600722138 +0000
+@@ -67,6 +67,7 @@
```

Comment:
Please add a comment indicating that this patch would no longer be necessary with scikit-image 0.18 (which includes https://github.com/scikit-image/scikit-image/pull/5059)